frontend/src/components/MapRenderer.ts

```diff
@@ -1,42 +1,86 @@
 import { Position } from 'geojson';
 import mapboxgl from 'mapbox-gl';
 import MapboxDraw from '@mapbox/mapbox-gl-draw';
-import { Annotations } from '../models/types';
+import { Annotations, Project } from '../models/types';
 
 // @ts-ignore
 // eslint-disable-next-line import/no-webpack-loader-syntax, import/no-unresolved
 mapboxgl.workerClass = require('worker-loader!mapbox-gl/dist/mapbox-gl-csp-worker').default;
 
-const BLUE = '#8FB1BB';
-const DARK_BLUE = '#1A85A7';
-const ORANGE = '#F29D49';
-const BEIGE = '#B4A88C';
-const LIGHT_BEIGE = '#D7CFBE';
+const SEARCH_ITEM_COLOUR = '#1A85A7'; // teal
+const SELECTED_ITEM_COLOUR = '#E28D38'; // orange
+
+// const LIGHT_BEIGE = '#D7CFBE';
```

Comment:
Delete stray comment?

Reply:
gonna make annotations beige in followup

Reply:
Sounds good! Thanks for making small PRs thats actually very much appreciated :)